pulumi/infra/consul_config.py

```
@@ -0,0 +1,83 @@
+                            "name": "zipkin",
+                            "type": "STRICT_DNS",
+                            "connect_timeout": "5s",
+                            "load_assignment": {
+                                "cluster_name": "zipkin",
+                                "endpoints": [
+                                    {
+                                        "lb_endpoints": [
+                                            {
+                                                "endpoint": {
+                                                    "address": {
+                                                        "socket_address": {
+                                                            "address": tracing_endpoint,
+                                                            "port_value": 9411,
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        ]
+                                    }
+                                ],
+                            },
+                        }
+                    ),
+                    "envoy_stats_flush_interval": "10s",
+                    "envoy_tracing_json": json.dumps(
+                        {
+                            "http": {
+                                "name": "envoy.tracers.zipkin",
+                                "typedConfig": {
+                                    "@type": "type.googleapis.com/envoy.config.trace.v3.ZipkinConfig",
+                                    "collector_cluster": "zipkin",
+                                    "collector_endpoint_version": "HTTP_JSON",
+                                    "collector_endpoint": "/api/v2/spans",
+                                    "shared_span_context": False,
+                                    "trace_id_128bit": True,
```

This is apparently the magic config required to get tracing working with Jaeger. Without it, this fails silently